bookkeeper-server/src/main/java/org/apache/bookkeeper/client/ExplicitLacFlushPolicy.java

```diff
@@ -111,7 +110,7 @@ public String toString() {
             };
             try {
                 scheduledFuture = lh.bk.mainWorkerPool.scheduleAtFixedRateOrdered(lh.getId(), updateLacTask,
-                        explicitLacIntervalInSec, explicitLacIntervalInSec, SECONDS);
+                        explicitLacIntervalInSec, explicitLacIntervalInSec, TimeUnit.MILLISECONDS);
```

Comment:
So the variable names say that they are in Seconds but the timeunit says to interpret it as milliseconds.  Please at least rename the variables.